postgresql-17.yaml

```diff
@@ -195,6 +200,68 @@ subpackages:
       provides:
         - libpq=${{package.full-version}}
 
+  - name: plpython3
+    description: PostgreSQL Python libraries
+    dependencies:
+      provider-priority: 17
+      provides:
+        - plpython=${{package.full-version}}
```

Comment:
I don't see a reason to introduce a `plpython` name. virtualpackages can cause mysterious issues with our resolver, so I prefer we only add them if we have a real use case. This just seems like an alias.